builder/azure/arm/step_power_off_compute.go

```diff
@@ -28,7 +29,7 @@ func NewStepPowerOffCompute(client *AzureClient, ui packersdk.Ui) *StepPowerOffC
 }
 
 func (s *StepPowerOffCompute) powerOffCompute(ctx context.Context, resourceGroupName string, computeName string) error {
-	f, err := s.client.VirtualMachinesClient.Deallocate(ctx, resourceGroupName, computeName)
+	f, err := s.client.VirtualMachinesClient.Deallocate(ctx, resourceGroupName, computeName, flag.Bool("forceDelete", true, "forces fast deletion"))
```

Comment:
```suggestion
	f, err := s.client.VirtualMachinesClient.Deallocate(ctx, resourceGroupName, computeName, false)
```
see https://github.com/hashicorp/packer-plugin-azure/pull/199#issuecomment-1135075590

Reply:
fixed